src/index.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ]  ESLint is a code analysis tool that is used to check the code quality of JavaScript code. Therefore I kindly suggest to avoid turning off the Eslint to ensure that your code is of high quality ☺️.